docker/nginx/config/project.conf

```diff
@@ -0,0 +1,38 @@
+server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
+	# Replace this with domain/ip you're running on
+        server_name www.example-server.com;
+        return 301 https://$server_name$request_uri;
+        # Required by Let's Encrypt
+        location ~ /.well-known {
+                allow all;
+        }
+	# Do not log visitor information
+        access_log off;
+}
+server {
+        # SSL configuration
+        #
+        listen 443 ssl http2;
+        listen [::]:443 ssl http2;
+	# If there is a letsencrypt cert, let us use this instead
+	# include snippets/ssl-params.conf;
+	# Otherwise, use hardcoded ssl cert for debugging purposes
+	# Can easily generate new ones following;
+	# https://www.digitalocean.com/community/tutorials/how-to-create-a-ssl-certificate-on-nginx-for-ubuntu-12-04
+	ssl on;
+	ssl_certificate /etc/nginx/conf.d/server.crt;
```

Comment:
is this path correct? It sticks out to me as not being wrong. `/etc/nginx/conf.d/server.crt` but I see this file path as `docker/nginx/config/server.crt`. unless this file `docker/nginx/Dockerfile` does the remapping of relative paths.